Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -138,6 +138,7 @@ class TrackingProtectionTests: BaseTestCase {
         // A page displaying the connection is secure
         XCTAssertTrue(app.staticTexts["mozilla.org"].exists)
         XCTAssertTrue(app.staticTexts["Connection is secure"].exists, "Missing Connection is secure info")
+        XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Secure connection")
```

Comment:
An extra assertion to ensure that the lock icon's label corresponds to the icon displayed. 🔒